Replace use of <net/if_packet.h> with <linux/if_packet.h>.

kernel headers <linux/if_packet.h> already provides the
needed definitions, moreover not all libc implementations
provide if_packet.h e.g. musl

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending
[From http://cgit.openembedded.org/openembedded-core/plain/meta/recipes-connectivity/irda-utils/irda-utils-0.9.18/musl.patch.]
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

Index: irda-utils-0.9.18/irdaping/irdaping.c
===================================================================
--- irda-utils-0.9.18.orig/irdaping/irdaping.c
+++ irda-utils-0.9.18/irdaping/irdaping.c
@@ -33,7 +33,6 @@
 #include <sys/socket.h>
 #include <sys/ioctl.h>
 #include <net/if.h>		/* For struct ifreq */
-#include <net/if_packet.h>	/* For struct sockaddr_pkt */
 #include <net/if_arp.h>		/* For ARPHRD_IRDA */
 #include <netinet/if_ether.h>	/* For ETH_P_ALL */
 #include <netinet/in.h>		/* For htons */
@@ -46,6 +45,7 @@
 #include <asm/byteorder.h>	/* __cpu_to_le32 and co. */
 
 #include <linux/types.h>	/* For __u8 and co. */
+#include <linux/if_packet.h>	/* For struct sockaddr_pkt */
 #include <irda.h>
 
 #ifndef AF_IRDA
